src/app/auth/services/user.service.ts

```diff
@@ -91,6 +91,12 @@ const checkChangeEmailLinkExpiration = (verifyToken: string): Promise<CheckChang
   return authClient.checkChangeEmailExpiration(verifyToken);
 };
 
+const downloadAvatar = async (url: string): Promise<Blob> => {
+  const response = await fetch(url);
+  const data = await response.blob();
+  return data;
+};
+
```

Comment:
This snippet is used at least in `AvatarWrapper`, I would check where to substitute this function where needed 🔍

Reply:
Indeed, I forgot to use this function in `AvatarWrapper`. Let me see if I can use it in more places.